dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/wss11sig/CxfWss11SigTests.java

```diff
@@ -308,7 +308,7 @@ public void testCXFClientBasicEncryptedElementMisMatchEE7Only() throws Exception
 
     @Test
     @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
-    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
+    @AllowedFFDC(value = { "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
```

Comment:
The exception `"org.apache.wss4j.common.ext.WSSecurityException"` is missed in @AllowedFFDC? or it's not needed?  This exception is usually used for EE8.

Reply:
the FFDC is not needed
i ran the test without it and it still passed
only the `java.net.MalformedURLException`  FFDC was thrown for that test

Reply:
ok thanks for clarification.